main.go

```diff
@@ -260,6 +260,23 @@ func AddNewAddress(ep *v1.EndpointSubset, address string, host string) {
 	ep.NotReadyAddresses = append(ep.NotReadyAddresses, newAddr)
 }
 
+func CheckDNSChange(ep *v1.EndpointSubset, ip string, hostname string) {
```

Comment:
What happens if we change some dns in the endpoint's annotation ? I think that currently we aren't removing old addresses.